docs/usage/self-hosted-configuration.md

```diff
@@ -852,6 +878,13 @@ It also may mean that ignored directories like `node_modules` can be preserved a
 
 ## platform
 
+## platformVersion
+
+Specify this string for Renovate to skip API checks and provide GitLab/Bitbucket server version directly.
```

Comment:
```suggestion
Set a string here to let Renovate skip some API checks.
Renovate will use this value as the GitLab or Bitbucket server version.
```

Maybe we should explain which API check(s) Renovate will skip?

I think the key point is that renovate skips a "fetch version" API call?